x/application/keeper/msg_server_stake_application.go

```diff
@@ -61,8 +62,9 @@ func (k msgServer) createApplication(
 	msg *types.MsgStakeApplication,
 ) types.Application {
 	return types.Application{
-		Address: msg.Address,
-		Stake:   msg.Stake,
+		Address:          msg.Address,
+		Stake:            msg.Stake,
+		DelegateePubKeys: make([]codectypes.Any, 0),
```

Comment:
Not a fan of the fact that we have to use `Any` in such a core piece of the protocol.

Reply:
`Any` is the cosmos standard way of doing pubkey encoding. Their `Any` type is a modified version of the protobuf message. Cosmos uses it internally for this exact feature which is why I am using it